### DISCUSSION TITLE
Welcome to playerctl Discussions!

### DISCUSSION THREAD

**Discussion by acrisci** [altdesktop/playerctl#210] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VastOne** [comment]
Hey Tony, brilliant code!

I just found it today when I began having some issues keeping a player playing and just wanted to say it is fantastic..Thank you
